frontend/src/metabase/visualizations/visualizations/Map.jsx

```diff
@@ -341,3 +352,26 @@ export default class Map extends Component {
     }
   }
 }
+
+const mapStateToProps = (state, props) => {
+  return {
+    isAdmin: getUserIsAdmin(state, props),
+  };
+};
+
+const CustomMapFooter = connect(mapStateToProps)(({ isAdmin }) => {
```

Comment:
This component won't get a proper displayName (there should be a warning in the console). Use something like `connect(..., function CustomMapFooter({ isAdmin }) {`